tools/scripts/fuzz.sh

```diff
@@ -1,21 +1,39 @@
 #!/bin/bash
 
-set -exuo pipefail
+set -euo pipefail
 
-# build the actual fuzz target
-bazel build //test/fuzz:fuzz_dash_e --config=fuzz -c opt
-PATH=$PATH:$(pwd)/bazel-sorbet/external/llvm_toolchain/bin/
-export PATH
-export ASAN_OPTIONS=dedup_token_length=10 # use top 10 frames to tell different errors appart
+what=""
+if [ "$#" -eq 0 ]; then
+  what="fuzz_dash_e"
+elif [ "$#" -eq 1 ]; then
+  what="$1"
+else
+  echo "usage: $0 [<fuzz_target>]"
+  exit 1
+fi
 
+echo "building $what"
+bazel build "//test/fuzz:$what" --config=fuzz -c opt
 
-# setup corpus
-mkdir -p fuzz_corpus
-find ./test/testdata/ -iname "*.rb"|grep -v disable|xargs -n 1 -I % cp % fuzz_corpus
+# normally we'd like to check to see if commands are around before running the script, but in this case, we want the
+# bazel build command to run before this check so that bazel can download itself.
+echo "checking for commands"
+export PATH="$PATH:$PWD/bazel-sorbet/external/llvm_toolchain/bin"
+if ! command -v llvm-symbolizer >/dev/null; then
+  echo "fatal: command not found: llvm-symbolizer"
+  exit 1
+fi
 
+echo "setting up files"
+mkdir -p fuzz_corpus
+find test/testdata -iname "*.rb" | grep -v disable | xargs -n 1 -I % cp % fuzz_corpus
 mkdir -p fuzz_crashers/original
 
-command -v llvm-symbolizer >/dev/null 2>&1 || { echo 'will need llvm-symbolizer' ; exit 1; }
-
-# fuzz
-nice ./bazel-bin/test/fuzz/fuzz_dash_e -use_value_profile=1 -only_ascii=1 -dict=test/fuzz/ruby.dict -artifact_prefix=fuzz_crashers/original/ fuzz_corpus/ --stress-incremental-resolver
+echo "running"
+export ASAN_OPTIONS="dedup_token_length=10"
```

Comment:
You should put the helpful comment back in (`# use top 10 frames to tell different errors apart`)